packages/create-liveblocks-app/bin/templates/init/config-generation.ts

```diff
@@ -140,7 +140,9 @@ function allHooks() {
   useRedo,
   useCanUndo,
   useCanRedo,
-  useMutation,`;
+  useMutation,
+  useStatus,
+  useLostConnectionListener,`;
```

Comment:
Just double-checking: will this also add it to the suspense block?

Reply:
> Just double-checking: will this also add it to the suspense block?

Yep!